guess/touchscreen/action

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -eu
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+. guess-functions
+
+$TRACE_SOURCE "Processing ..."
+
+# The idea comes from tslib. Touchscreen devices have
+# the INPUT_PROP_DIRECT property, which is determined
+# by the first from zero bit of the property number
+
+INPUT_PROP_DIRECT=1
+bitmask=$((1 << $INPUT_PROP_DIRECT))
+
+for i in "$SYSFS_PATH"/class/input/event*/device/properties; do
+	readline prop "$i"
```

Comment:
Before reading a file, you need to check that the `$i` exists.

```diff
 for i in "$SYSFS_PATH"/class/input/event*/device/properties; do
+    [ -f "$i" ] ||
+        continue
     readline prop "$i"
```
If there are no files in sysfs that satisfy this fnmatch, then `$i` will contain the pattern itself `/sys/class/input/event*/device/properties`.

Reply:
I guessed that `$i` will be empty string and 'for' cycle will not be executed. Thanks